lib/magma/commands.rb

```diff
@@ -2,6 +2,42 @@
 require 'logger'
 
 class Magma
+  class LoadProject < Etna::Command
+    def execute(file_name)
+      file = File.open(file_name)
+      file_data = JSON.parse(JSON.parse(file.read))
+
+      db = Magma.instance.db
+
+      models = file_data["models"]
+
+      models.keys.each do |model|
+        model_name = models[model]["template"]["name"]
+        attributes = models[model]["template"]["attributes"]
+
+        attributes.keys.each do |att|
+          attribute = attributes[att]
+
+          db[:attributes].insert(
+            project_name: 'i dont know where to find the project name',
+            model_name: model_name,
+            attribute_name: attribute["name"],
+            type: attribute["type"] || "project",
```

Comment:
not sure I understand the `|| "project"` here - there shouldn't be a default for "type"?

Reply:
This was a placeholder. In the JSON I retrieved for this test, the `project` doesn't have a `type`: 
![image](https://user-images.githubusercontent.com/15822772/83428106-0d11c100-a3ef-11ea-8b20-cc7d59e6240e.png)